distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -9,62 +9,37 @@ body {
   color: $normal-text;
   font-family: helvetica, sans-serif;
   margin: 0;
-
-  &:not(.filtered) {
-    padding: 0 $default-whitespace;
-  }
 }
 
 a {
   color: $mild-blue;
 }
 
 .global-wrapper {
+  display: flex;
+  flex-wrap: wrap;
   margin: 0 auto;
-  max-width: 1400px;
   padding: 10px;
-
-  .filtered & {
-    padding: 10px 0 0 $filters-width;
-  }
+  width: 100%;
 }
 
 .filtered-charts {
   display: flex;
-
-  @media all and (max-width: $min-sidebar-width) {
-    display: block;
-  }
 }
 
 .filters {
+  display: none;
   font-weight: 100;
   height: 100%;
-  left: 0;
-  padding: $header-height 20px 20px 0;
-  position: absolute;
-  top: 0;
+  padding: 40px 20px 20px 0;
+  margin-left: -10px;
```

Comment:
To ignore the `.global-wrapper` padding (hover effect should touch the left edge). I should leave a comment though.